h2/utilities.py

```diff
@@ -329,6 +329,16 @@ def _reject_pseudo_header_fields(headers, hdr_validation_flags):
             seen_pseudo_header_fields
         )
 
+    # If ':status' pseudo-header is not there in a response header, reject it
```

Comment:
It would be nice if this comment had a reference to the exact RFC section – I know we‘ve not been super consistent on that point, but I think it’s helpful.

To save you looking it up again: `RFC 7540 § 8.1.2.4`.

https://tools.ietf.org/html/rfc7540#section-8.1.2.4